### PR TITLE
Add documentation for acaia integration

### PR DESCRIPTION
**Pull request by zweckj**
## Proposed change

SSIA

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/130059
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/6075
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced documentation for integrating Acaia smart coffee scales with Home Assistant.
	- Added support for Bluetooth connectivity for automatic scale discovery.

- **Documentation**
	- Included detailed setup instructions, automation examples, and troubleshooting advice for users.
	- Highlighted functionalities for button operations like tare, reset timer, and start/stop timer.
	- Listed supported devices and encouraged user feedback for additional models.
	- Addressed known limitations and provided guidance on common issues.

### CODE REVIEW

File: source/_integrations/acaia.markdown
Comment (zweckj) [reply source/_integrations/acaia.markdown:4]: current release is 2024.11, 2014.12 is the next release

File: source/_integrations/acaia.markdown
Comment (c0ffeeca7) [inline source/_integrations/acaia.markdown]:
```suggestion
The **Acaia** {% term integrations %} allows you to control [Acaia](https://acaia.co/) scales through Home Assistant.
```

File: source/_integrations/acaia.markdown
Comment (c0ffeeca7) [inline source/_integrations/acaia.markdown]:
```suggestion
If your machine is within Bluetooth range to your Home Assistant host and the [Bluetooth](/integrations/bluetooth) integration is fully loaded, the scale should be discovered automatically.
```

File: source/_integrations/acaia.markdown
Comment (c0ffeeca7) [inline source/_integrations/acaia.markdown]:
```suggestion
description: "When a brew starts on the machine, the following actions are started: tare, reset the timer, and start the timer on the scale."
```

File: source/_integrations/acaia.markdown
Comment (c0ffeeca7) [inline source/_integrations/acaia.markdown]:
```suggestion
- **Tare**: Tares the scale.
```

File: source/_integrations/acaia.markdown
Comment (c0ffeeca7) [inline source/_integrations/acaia.markdown]:
```suggestion
This integration can be used in combination with integrations for smart coffee machines, such as the [La Marzocco integration](https://www.home-assistant.io/integrations/lamarzocco/) integration.
```

File: source/_integrations/acaia.markdown
Comment (c0ffeeca7) [inline source/_integrations/acaia.markdown]:
```suggestion
It could also be used to display the weight on secondary displays when brewing on a Pyxis or Lunar where you cannot see the display.
```

**Review by c0ffeeca7 — COMMENTED** [review]
Some tweaks

File: source/_integrations/acaia.markdown
Comment (c0ffeeca7) [inline source/_integrations/acaia.markdown]:
```suggestion
```

File: source/_integrations/acaia.markdown
Comment (c0ffeeca7) [inline source/_integrations/acaia.markdown]:
```suggestion
```
missed that one before. the type doesn't apply in configuration_basic

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @zweckj :+1:

File: source/_integrations/acaia.markdown
Comment (c0ffeeca7) [inline source/_integrations/acaia.markdown]:
```suggestion
```

File: source/_integrations/acaia.markdown
Comment (c0ffeeca7) [inline source/_integrations/acaia.markdown]:
```suggestion
```
Still got it wrong. The "required" doesn't apply here either, as the UI will take care of that.

**Review by c0ffeeca7 — DISMISSED** [review]

**Comment by zweckj** [comment]
Sorry @c0ffeeca7, I was just doing some last changes...

**Comment by c0ffeeca7** [comment]
> Sorry @c0ffeeca7, I was just doing some last changes...

Haha, nothing to apologize for. :-D It's on next, so changes are to be expected. Usually, I wait until the parent is merged. But I wanted to get ahead a bit.

File: source/_integrations/acaia.markdown
Comment (c0ffeeca7) [inline source/_integrations/acaia.markdown]:
```suggestion
{% details "Problem: Not reacting to commands" %}
```
what is not reacting? the scale?

File: source/_integrations/acaia.markdown
Comment (c0ffeeca7) [inline source/_integrations/acaia.markdown]: and if it is a newer model, how does that help?

File: source/_integrations/acaia.markdown
Comment (c0ffeeca7) [inline source/_integrations/acaia.markdown]:
```suggestion
Make sure your scale is in Bluetooth range to your Home Assistant instance. [ESPHome Bluetooth Proxies](https://esphome.io/components/bluetooth_proxy.html) are a great way to increase the range if your instance is too far away.
```

File: source/_integrations/acaia.markdown
Comment (zweckj) [reply source/_integrations/acaia.markdown]: actually, I'll move that

**Review by c0ffeeca7 — DISMISSED** [review]
Nice! I like the details sections.

**Comment by zweckj** [comment]
@c0ffeeca7 thought I was done, but found a bug with certain devices that I wanted to document. Should be really done this time 😅

File: source/_integrations/acaia.markdown
Comment (frenck) [inline source/_integrations/acaia.markdown]:
```suggestion
```

File: source/_integrations/acaia.markdown
Comment (frenck) [inline source/_integrations/acaia.markdown]:
```suggestion
alias: "Start timer on scale"
```

File: source/_integrations/acaia.markdown
Comment (frenck) [inline source/_integrations/acaia.markdown]:
```suggestion
  - trigger: state
    entity_id:
      - binary_sensor.lm001234_brewing_active
    to: "on"
    from: "off"
```

File: source/_integrations/acaia.markdown
Comment (frenck) [inline source/_integrations/acaia.markdown]:
```suggestion
```

File: source/_integrations/acaia.markdown
Comment (frenck) [inline source/_integrations/acaia.markdown]:
```suggestion
```

File: source/_integrations/acaia.markdown
Comment (frenck) [inline source/_integrations/acaia.markdown]:
```suggestion
```

**Review by frenck — APPROVED** [review]
Thanks, @zweckj 👍 

../Frenck

**Pull request merged**
